UnoCheck/CheckCommand.cs

```diff
@@ -1,327 +1,355 @@
+            Util.Verbose = settings.Verbose;
+            Util.LogFile = settings.LogFile;
+            Util.CI = settings.CI;
+            if (settings.CI)
+                settings.NonInteractive = true;
+
+            Console.Title = ToolInfo.ToolName;
+
+            AnsiConsole.Markup(AsciiAssets.UnoLogo);
+            AnsiConsole.WriteLine();
+            AnsiConsole.Render(
+                new FigletText("uno-check").LeftAligned().Color(new Color(122, 103, 247)));
+
+            var currentVersion = ToolInfo.CurrentVersion;
+            AnsiConsole.MarkupLine($"[underline bold green]{Icon.Ambulance} {ToolInfo.ToolName} v{currentVersion} {Icon.Recommend}[/]");
+            AnsiConsole.Render(new Rule());
+
+            AnsiConsole.MarkupLine("This tool will check your Uno Platform development environment.");
+            AnsiConsole.MarkupLine("If problems are detected, it will offer the option to try and fix them for you, or suggest a way to fix them yourself.");
+            AnsiConsole.Render(new Rule());
 
-			Console.Title = ToolInfo.ToolName;
 
-			AnsiConsole.Markup(AsciiAssets.UnoLogo);
-			AnsiConsole.WriteLine();
-			AnsiConsole.Render(
-				new FigletText("uno-check").LeftAligned().Color(new Color(122, 103, 247)));
+            NuGetVersion latestVersion = null;
+            try
+            {
+                latestVersion = await ToolInfo.GetLatestVersion(currentVersion.IsPrerelease);
+            }
+            catch
+            {
+                AnsiConsole.MarkupLine($"[bold yellow]{Icon.Warning} Could not check for latest version of uno-check on NuGet.org. The currently installed version may be out of date.[/]");
+                AnsiConsole.WriteLine();
+                AnsiConsole.Render(new Rule());
+            }
 
-			AnsiConsole.MarkupLine($"[underline bold green]{Icon.Ambulance} {ToolInfo.ToolName} v{ToolInfo.CurrentVersion} {Icon.Recommend}[/]");
-			AnsiConsole.Render(new Rule());
+            if (latestVersion is not null && currentVersion < latestVersion)
+            {
+                AnsiConsole.MarkupLine($"[bold yellow]{Icon.Warning} Your uno-check version is not up to date. The latest version is {latestVersion}. You can use the following command to update:[/]");
+                AnsiConsole.WriteLine();
+                AnsiConsole.MarkupLine($"dotnet tool update --global Uno.Check --version {latestVersion}");
+                AnsiConsole.WriteLine();
 
-			AnsiConsole.MarkupLine("This tool will check your Uno Platform development environment.");
-			AnsiConsole.MarkupLine("If problems are detected, it will offer the option to try and fix them for you, or suggest a way to fix them yourself.");
-			AnsiConsole.Render(new Rule());
+                AnsiConsole.Render(new Rule());               
+            }
 
-			if (!Util.IsAdmin() && Util.IsWindows)
-			{
-				var suTxt = Util.IsWindows ? "Administrator" : "Superuser (su)";
+            if (latestVersion is null || currentVersion < latestVersion)
```

Comment:
We're likely need to skip this if we're specifying a manifest on the CLI and/or that we're running with the `--ci` flag.

Reply:
Done!